### ISSUE TITLE
HTML is hiding under Navigation bar. 

### ISSUE THREAD

**Issue by amrit42087** [nnhubbard/ZSSRichTextEditor#150]
I am using the editor in iPad Landscape mode. 
Please see the image attached
![screen shot 2016-11-16 at 3 37 06 pm](https://cloud.githubusercontent.com/assets/11562827/20342988/b19fb714-ac12-11e6-9b51-079ace52fd6d.png)

**Comment by nnhubbard** [comment]
This is correct behavior for a `UINavigationController` after iOS 7. If you don't want the content to go under the `navigationBar`, you need to make sure to set `navigationController.navigationBar.translucent = NO;`.
